python/mhs/cf_common.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (c) 2019, GEM Foundation.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.
+# If not, see <https://www.gnu.org/licenses/agpl.html>.
+#
+
+import sys
+
+from django.db import connections
+
+
+class License():
+    """
+    License codes and ids present in DB
+    """
+    _LICENSE_ID_QUERY = "SELECT id, code FROM cf_common.license"
+    _LICENSE_DICT = {}
+
+    "Call once to load licenses into memory"
+    @classmethod
+    def load_licenses(cls, cursor):
+        cursor.execute(License._LICENSE_ID_QUERY)
+        for row in cursor.fetchall():
+            License._LICENSE_DICT[row[1]] = row[0]
+        print "Load licenses - loaded: {0}".format(License._LICENSE_DICT)
+
```

Comment:
If you get in the habit of putting parenthesis around print it will help your future transaction to Python 3 (they work in Python 2 too for your usage).